src/components/treemap.tsx

```diff
@@ -1,11 +1,10 @@
-import {
-  treemap,
-  treemapResquarify,
-} from 'd3';
+import { treemap, treemapResquarify } from 'd3';
```

Comment:
How come one line? Did the linter say something?
I usually start doing a vertical list if there is more than one, except for certain cases where I know that there will only be 2 items brought in from a module.